README.md

```diff
@@ -93,7 +93,6 @@ has instructions for running the samples.
 
 | Sample                      | Source Code                       | Try it |
 | --------------------------- | --------------------------------- | ------ |
-| List Recommendations | [source code](https://github.com/googleapis/java-recommender/blob/master/samples/generated/src/main/java/com/google/cloud/examples/recommender/v1beta1/SampleListRecommendations.java) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/java-recommender&page=editor&open_in_editor=samples/generated/src/main/java/com/google/cloud/examples/recommender/v1beta1/SampleListRecommendations.java) |
```

Comment:
This file is generated. If we actually want to delete this entry, we need to delete the sample in the link and automation can remove this from the README.